package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "types": "index.d.ts",
   "scripts": {
-    "docs": "./docs/scripts/build_docs.sh apm-agent-nodejs ./docs ./build",
+    "docs:open": "PREVIEW=1 ./docs/scripts/build_docs.sh apm-agent-nodejs ./docs ./build",
```

Comment:
I believe you could also do `PREVIEW=1 npm run docs:build` to reduce duplication.

Reply:
oooo I like that! Let me try

Reply:
Nice, it works!